mod_tariffs.asciidoc

```diff
@@ -131,17 +131,18 @@ Example:
 |===
 |Method |Description 
 
-|<<mod_tariffs_msp_get_method,GET>> |Retrieve a Tariff as it is stored in the eMSP system. 
+|<<mod_tariffs_msp_get_method,GET>> |Retrieve a Tariff as it is stored in the eMSP's system. 
 |POST |n/a 
 |<<mod_tariffs_put_method,PUT>> |Push new/updated Tariff object to the eMSP. 
 |PATCH|n/a
-|<<mod_tariffs_delete_method,DELETE>> |Remove Tariff object which is no longer valid 
+|<<mod_tariffs_delete_method,DELETE>> |Remove a Tariff object which is no longer valid. 
```

Comment:
Well, I see two messages behind this action:
- CPO is not going to use the Tariff any longer (when billing the EMP).
- EMP should not base his tariffs on this Tariff any longer, as it is not valid any more.

Therefore, I think _no longer valid_ is not too bad of a wording. Implementations of Tariffs may also use a `valid_from` and `valid_until` field structure, making the wording even more perfect.

Reply:
The CPO can send a list of Tariffs, to a eMSP. The Locations `Connector` object links to `tariff_ids`. So there might be Tariffs send to the eMSP that are not used in any `Connector`. Such a Tariff is then still valid, but not used. By adding wording to clarify this, we might prevent questions from implementers that could now be in doubt: May I remove a tariff that is still valid, `end_date_time` is not set or has a value in the future?

Reply:
Does this mean you may not delete a Tariff until all references of Connectors to the Tariff are removed?

Reply:
We have not defined that rule, but I think it might be good if we add some remark about that. If a location has a tariff_id that is not known to the eMSP that that might not be a good idea. Do we need a strict rule? I feel more for an advice that the CPO should check if the Tariff is not used in a Connector before deleting it.

Reply:
I changed the wording from _valid_ to _used_ and added a note about the references in bc71634.